### PR TITLE
#36 - design of a company site

### PR DESCRIPTION
**Pull request by paulinaszkudlarek**
image
image
image

Mobile:
image

image
image
image

### CODE REVIEW

File: frontend/components/Company/CompanyMap.vue [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<script setup>
+import CompaniesMap from '@/components/Companies/Map/CompaniesMap.vue'
+
+defineProps({
+  company: {
+    type: Object,
+    default: () => { },
```
Comment (EwelinaSkrzypacz) [inline frontend/components/Company/CompanyMap.vue]:
```suggestion
    default: () => {},
```

File: frontend/components/Company/ListTabs.vue [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  return lists.value
+})
+
+</script>
+
+<template>
+  <div class="my-10 bg-white text-center text-lg font-bold tracking-tight text-zinc-700 shadow-lg md:text-2xl">
+    <div class="lg:py-18 mx-auto block w-full max-w-7xl p-4 sm:px-6 lg:px-8">
+      <h2 class="first-letter:text-laravel lg:py-18 w-full justify-center border-b py-6 px-7 text-left text-2xl font-bold sm:px-6 md:text-4xl lg:px-8"> 
+        Meetup'owe statystyki
```
Comment (EwelinaSkrzypacz) [inline frontend/components/Company/ListTabs.vue]: I'm not sure if "statystyki" here is a good word.

File: frontend/components/Company/ListTabs.vue [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+                selected
+                  ? 'bg-white shadow'
+                  : 'text-zinc-100 hover:bg-white/[0.12] hover:text-white'
+              ]"
+              >
+                {{ category.name }}
+              </button>
+            </Tab>
+          </TabList>
+
```
Comment (EwelinaSkrzypacz) [inline frontend/components/Company/ListTabs.vue]:
```suggestion
```

File: frontend/components/Company/ListTabs.vue [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+                    <h3 v-if="elem[1].title" class="text-left font-medium leading-5">
+                      {{ elem[1].title }}
+                    </h3>
+                    <div v-else class="flex items-center space-x-5">
+                      <img v-if="elem[1].image" :src="elem[1].image" class="h-10 w-10 rounded-full hover:scale-110">
+                      <h3 class="text-left font-medium leading-5">
+                        {{ elem[1].name }}
+                      </h3>
+                    </div>
+
```
Comment (EwelinaSkrzypacz) [inline frontend/components/Company/ListTabs.vue]: Do we need this extra empty lines?

**Review by EwelinaSkrzypacz — COMMENTED** [review]
1. For UX it will be better to not center text - left alignment is okay.
2. It looks like these icons are crooked

image

File: frontend/components/Company/ListTabs.vue (hunk 4 above)
Comment (paulinaszkudlarek) [reply frontend/components/Company/ListTabs.vue]: the code is clearer, but no pressure

File: frontend/components/Company/ListTabs.vue (hunk 2 above)
Comment (paulinaszkudlarek) [reply frontend/components/Company/ListTabs.vue]: maybe "Udział w Meetup'ach" or "Meetup'owy wkład"?

Comment (EwelinaSkrzypacz) [reply frontend/components/Company/ListTabs.vue]: "Meetupowy wkład" will be okay.

File: frontend/pages/Companies.vue [hunk 5]
```diff
@@ -23,5 +23,5 @@ onMounted(async () => {
   <page-header word1="Organizatorzy" word2="i" word3="partnerzy" sentence="Dołącz do nas i promój swoją firmę!"/>
```
Comment (EwelinaSkrzypacz) [inline frontend/pages/Companies.vue]: Please change to "promuj"

File: frontend/pages/Company.vue [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+  })
+  async function fetchMeetups () {
+    const response = await fetch('/api/meetups.json')
+    meetups.value = await response.json()
+    return meetups
+  }
+  fetchMeetups().then(meetups => {
+    meetups
+  })
+
```
Comment (EwelinaSkrzypacz) [inline frontend/pages/Company.vue]:
```suggestion
```

**Review by EwelinaSkrzypacz — COMMENTED** [review]
It's just suggestion - on smaller resolutions maybe make select input to select tab, I mean something like [here](https://tailwindui.com/components/application-ui/navigation/tabs#component-83b472fc38b57e49a566805a5e5bb2f7).



image

**Review by EwelinaSkrzypacz — DISMISSED** [review]

**Review by MichalMyskow — APPROVED** [review]

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Pull request merged**